src/ucp/wireup/address.c

```diff
@@ -677,14 +686,17 @@ ucp_address_pack_iface_attr_v2(ucp_worker_h worker, void *ptr,
     unsigned seg_size                          = 0;
     ucp_address_v2_packed_iface_attr_t *packed = ptr;
     uint64_t addr_iface_flags;
+    double latency_nsec, overhead_nsec;
+
+    latency_nsec  = ucp_tl_iface_latency(worker->context, &iface_attr->latency) *
+                    UCS_NSEC_PER_SEC;
+    overhead_nsec = iface_attr->overhead * UCS_NSEC_PER_SEC;
 
-    packed->overhead  = UCS_FP8_PACK(OVERHEAD, iface_attr->overhead);
+    packed->overhead  = UCS_FP8_PACK(OVERHEAD, overhead_nsec);
     packed->bandwidth = UCS_FP8_PACK(BANDWIDTH,
                                      ucp_tl_iface_bandwidth(worker->context,
                                      &iface_attr->bandwidth));
-    packed->latency   = UCS_FP8_PACK(LATENCY,
-                                     ucp_tl_iface_latency(worker->context,
-                                                          &iface_attr->latency));
+    packed->latency   = UCS_FP8_PACK(LATENCY, latency_nsec);
```

Comment:
pls add gtest to compare packed/unpack iface attributes (flag,s perf, segsize) with v2